dotnet/src/AutoGen/Core/HttpScheme.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// HttpScheme.cs
+
+namespace AutoGen;
+/// <summary>
+///    Ability to specify the protocol to use for the API
+/// </summary>
+public enum HttpScheme
```

Comment:
You can use `Uri.UriSchemHttps` instead of  creating your own scheme

doc: https://learn.microsoft.com/en-us/dotnet/api/system.uri.urischemehttps?view=net-8.0